free-courses-en.md

```diff
@@ -520,6 +521,7 @@
 * [Learn to Program Using Python](https://www.edx.org/course/learn-program-using-python-utarlingtonx-cse1309x) (edX)
 * [Programming Foundations with Python](https://www.udacity.com/course/programming-foundations-with-python--ud036) (Udacity)
 * [Python 3](https://www.sololearn.com/Course/Python/) (SoloLearn)
+* [Python And Cryptocurrency: Build 5 Real World Applications](https://www.udemy.com/course/coinmarketcap/) Ian Annase (Udemy)
```

Comment:
Could you please include a hyphen before the author?
Small change in this case, but it's always favorable to use the title from the source. Figured I'd put that forward as well since a change is required anyway.

```suggestion
* [Python & Cryptocurrency: Build 5 Real World Applications](https://www.udemy.com/course/coinmarketcap/) - Ian Annase (Udemy)
```